packages/flutter/lib/src/painting/notched_shapes.dart

```diff
@@ -37,10 +37,15 @@ abstract class NotchedShape {
 ///
 ///  * [CircleBorder], a [ShapeBorder] that describes a circle.
 class CircularNotchedRectangle extends NotchedShape {
+  /// A convex notch shape. It's concave by default.
+  final bool convex;
+
   /// Creates a [CircularNotchedRectangle].
   ///
+  /// `convex` Inverts the default concave shape to a convex shape.
```

Comment:
What does concave and convex mean in this context here? Is the convex option technically a notched shape?

Reply:
This means that one notch shape goes inside (concave) and the other goes outside (convex). But we could also name it just "invert". Or create a new shape without options. What do you think? I thought to invert the path Y direction by an option will reduce repetition code. This is just a proposal. :)